capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -72,7 +72,7 @@ class ClubPage extends Component {
       .catch(e => console.error(e));
 
     let members = [];
-    Promise.all(club.memberIDs.map(m => 
+    Promise.all(club.memberIDs.map(m =>
```

Comment:
`Promise.all` isn't needed unless you're `await`ing it or chaining a `.then` off of it: you'll need one of those if you want to ensure that all the async calls have completed before moving on

Reply:
In this case, since I'm chaining a `.then` off the `Promise.all` should I leave it as is?